dashboard/db/migrate/20180524174518_add_columns_to_script.rb

```diff
@@ -0,0 +1,8 @@
+class AddColumnsToScript < ActiveRecord::Migration[5.0]
```

Comment:
Do we need to extract the migration and ship it first?

Reply:
Sorry, yes this is separated out in https://github.com/code-dot-org/code-dot-org/pull/22689